rpc/rpcclientmap_test.go

```diff
@@ -18,13 +18,14 @@
 package rpc
 
 import (
-	"github.com/stretchr/testify/assert"
 	"math/rand"
 	"runtime"
 	"strconv"
 	"sync"
 	"testing"
 	"time"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
move out and place it into the 2rd import block.

Reply:
GC (CMS Initial Mark